airbyte-ci/connectors/pipelines/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
Instead, bump the version of this package in `pyproject.toml` file, and add changelog to the readme